README.md

```diff
@@ -1,2 +1,13 @@
 # uuid-gdb
 GDB pretty-printers for uuid::Uuid
+
+## Installation
+Add these lines to the `~/.gdbinit` file:
+```
+python
+import sys
+sys.path.insert(0, '/path/to/uuid-gdb')
+from uuid_printer import register_uuid_printer
+register_uuid_printer()
+end
+```
```

Comment:
maybe split the imports and register steps (`~/.gdbinit` may already have some of the imports)

Reply:
I modelled it after this one: https://github.com/ruediger/Boost-Pretty-Printer 

Do you mean a split with a linebreak?